redhat-cop/tower_utilities
# Fix rabbitmq order of comparaison and add variable to defaults

### What does this PR do?

Switch sorting of rabbitmq comparaison and add variable to defaults
    
the rabbitmq_required was comparing first with a version number then with 'latest', but 'latest'
can't be compared with a version, so it would fail before going to the 2nd step. Turning this
around solved the issue.
I also added the 'tower_release_version' variable to the defaults, where it was missing. 'latest' is the default, hope this is OK, for most demo/learning purposes, it's good enough I think.

And while I was at it, I've add Vim swap files to gitignore (bad habit I know).

### How should this be tested?

Install Tower with the 'latest' version resp. a fixed version and see that it doesn't fail any more.

### Is there a relevant Issue open for this?

n/a

### Other Relevant info, PRs, etc.

n/a
